### PR TITLE
Global styling framework: streamline variable handling

### PR DESCRIPTION
**Pull request by bryk**
Now variables are in variables.css in frontend directory. Previously they were in assets, which was wrong because it caused copy to production image.

### CODE REVIEW

**Comment by bryk** [comment]
@zreigz Can you review?

**Comment by zreigz** [comment]
yes

**Comment by floreks** [comment]
Just had a quick look. One comment about variable file naming. I've intentionally used underscore there.

http://sass-lang.com/guide

`Partials` headline says:
`The underscore lets Sass know that the file is only a partial file and that it should not be generated into a CSS file. Sass partials are used with the @import directive.`

Rest I'll leave for @zreigz to review :)

File: src/app/frontend/variables.scss [hunk 1]
```diff
@@ -20,6 +19,8 @@ $hover-primary: #1254df;
...
 $emphasis: #000;
 $content-background: #fff;
-// TODO(bryk): Reference angular_material/src/core/style/variables.scss instead of having
-// this variable here.
-$whiteframe-shadow-z1: 0 2px 5px 0 rgba(0, 0, 0, .26) !default;
+
+// TODO(bryk): Get those variables from Angular Material scss files.
+$layout-breakpoint-lg: 1280px;
+$whiteframe-shadow-1dp: 0 1px 3px 0 rgba(0, 0, 0, .2), 0 1px 1px 0 rgba(0, 0, 0, .14), 0 2px 1px -1px rgba(0, 0, 0, .12);
+$baseline-grid: 8px;
```
Comment (zreigz) [inline src/app/frontend/variables.scss]: For me ```$baseline-grid``` is misleading. I would suggest ```baseline-distance``` instead. What do you thing ?

**Comment by zreigz** [comment]
Rest looks good for me. All views look like previously.

File: src/app/frontend/variables.scss (hunk 1 above)
Comment (bryk) [inline src/app/frontend/variables.scss]: I'm also not a big fan of this name, but this is what Angular Material uses. Take a look at: https://github.com/angular/material/blob/HEAD/src/core/style/variables.scss

I hope that soon we'll be able to include angular material scss, so then this variable here will be deleted. And Material's one used. What do you think? Can I leave the name as it is?

**Comment by bryk** [comment]
@floreks Thank you for the comment! I renamed the file

**Comment by bryk** [comment]
PTAL

**Comment by zreigz** [comment]
If it is common practice to use name ```$baseline-grid``` then LGTM

**Pull request merged**